wordpress-mobile/WordPress-iOS
# Issues/5594 removes dtcoretext

Closes #5594 
This PR removes the DTCoreText pod from the project and switches to our own methods where DTCoreText methods were being used.

To test:
Switch to the branch and update your pods.
- Check the sites comments view controller. Confirm that comment cells show their content with whitespace normalized to a single space.
- Confirm new test passes.
- Check search bars in My Sites and the Reader.  Confirm the search bar's search icon, and cancel icon are correctly tinted. 
- Check Plans.  Confirm there is still a space between the plan title and the "Current Plan" marker.

Needs review: Santa Claus
